src/itemActionTitle/ItemActionTitle.story.mdx

```diff
@@ -0,0 +1,71 @@
+import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs/blocks'
+
+import { BubbleIcon } from '../icon/bubbleIcon'
+import { ItemActionTitle } from './index'
+
+<Meta title="Items/ItemActionTitle" />
+
+# ItemActionTitle
+
+### Button
+
+<Canvas>
+  <Story name="Button">
+    <ItemActionTitle
+      titleLabel="Title: primary information"
+      subLabel="Subtitle: secondary information"
+      action="Action"
+    />
+  </Story>
+</Canvas>
+
+### Link with leftAddon
+
+<Canvas>
+  <Story name="Link witn leftAddon">
+    <ItemActionTitle
+      titleLabel="Title: primary information"
+      subLabel="Subtitle: secondary information"
+      action="Action"
```

Comment:
nit: you could add a longer "Action" text. Like "Complete Action"

Reply:
Done